sponsor.html

```diff
@@ -0,0 +1,216 @@
+---
+layout: page
+title: Sponsor Us
+permalink: /sponsor/
+---
+<div class="panel panel-header panel-sponsor">
+  <div class="hero-container panel-container">
+    <div class="fixed-width">
+      <div class="hero-text">
+        <h1 class="header">Help us fund the future of Student Robotics</h1>
+        {% include donate.html %}
+      </div>
+    </div>
+  </div>
+</div>
+<div class="panel">
+  <div class="panel-container">
+    <div class="fixed-width">
+      <h2 class="text-center">Become a Sponsor</h2>
+      <div class="row text-center">
+	<p>
+          Student Robotics is looking for new sponsors to help us fulfil our mission.
+          The support from our sponsors will make it possible to sustain our programme
+          and continue to run free to enter competitions that attract students from
+          a wide range of schools and colleges. By partnering with us, you will be
+          able to increase your organisation or product awareness with the next
+          generation of engineers.
+	</p>
+	<p>
+	 Every year it costs us more than £20,000 to run the competition, and we
```

Comment:
This feels like it would be going into too much detail. Supporting the teams is part of running the competition, and the support will be subject to what the current competition team decide (which for a given year could be £0)

Reply:
@RealOrangeOne did a different thing. Different thing works

Reply:
Fixed in https://github.com/srobo/website/pull/283/commits/7df4f5273ec6fc255c9cbc3eea37300bdc48055a